### ISSUE TITLE
Gleev Logo

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/atlas#3276]
## Context
* https://github.com/Joystream/atlas/issues/3273

## Scope
* [ ] Create logo for Gleev
~Create logo for Vinteo~

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201958687417145/1203133549882145) by [Unito](https://www.unito.io)

**Comment by toiletgranny** [comment]
@bedeho, @dmtrjsg Gleev logo ideas are ready to be reviewed! :) Find out all about it from:

- [This Loom video update](https://www.loom.com/share/f01d05b32f4649f5a4016a73ef19cbf3) (10 min),
- [This Figma presentation](https://www.figma.com/proto/RBKQyC2bs8VWqm0mkY4kJv/Gleev-brand?page-id=2%3A22566&node-id=8%3A18296&viewport=-6752%2C-16%2C0.24&scaling=min-zoom&starting-point-node-id=8%3A18296).

cc @kdembler

**Comment by bedeho** [comment]
1. Thank you for the wonderful presentation, just love these, they are works of art in of themselves <3 
2. I love the analysis, I was thinking of this myself: the _atlas_ brand somehow was more core or closer to Joystream itself, Gleev is much more distant, and the last version you found really was able to show an example of something still in harmony with the rest of the product style, however quite independent from the core Joystream branding. This is exactly what we want, and exactly what we anticipate other operators will also want. They want to build a direct connection with the users through their own brand, not the Joystream brand.

I think all three were great, but #3 was the most powerful one, it also looked sweet in silhouette style.

**Comment by dmtrjsg** [comment]
Number 3 omg this is mindblowing :exploding_head:  and I  LOOOOVE it :heart_eyes: 
!!

**Comment by drillprop** [comment]
My girlfriend said that the silhouette version looks like a chess jumper and I can't get rid of this comparison from my mind 😅 The image of a chess jumper just stuck in my brain now. 
![image](https://user-images.githubusercontent.com/51168865/194703652-e7abb92b-aaeb-48b6-bdf7-df7aa3e36998.png)

**Comment by drillprop** [comment]
Don't get me wrong, I like the third one the most as well. Small suggestion -  how it would be like if we cut the part at the top(just a bit)? Up here 👇🏻 
![Zrzut ekranu 2022-10-8 o 13 44 31](https://user-images.githubusercontent.com/51168865/194705799-c80bf458-1a5e-408a-ae51-f261cbab80fd.png)

**Comment by bedeho** [comment]
🤣 Took me a while to even perceive this, doesn't stick with me.

**Comment by toiletgranny** [comment]
Ah, yes, been wondering how fast someone will discover it! The knight in the negative space is a byproduct of positioning the “G” letter at a certain angle. I started noticing this very early on, and while I tried to fight it initially (below you can see just a few samples), after a while, I realized I actually liked it, as it makes the icon more unique and interesting. So I left it there and took care of the negative space shape looking equally nice and proportional as the outer boundaries of the icon.

![image](https://user-images.githubusercontent.com/35307309/194814060-c52047fc-fb69-4886-9a60-458ba49e8a06.png)

**Comment by toiletgranny** [comment]
Alright, glad you guys liked it! Moving it back to the In Design column to have the new logo finalised and to move forward with updating all the assets!

**Comment by dmtrjsg** [comment]
On the call just now confirmed to finalise it in the shape and form that was in the original presentation.

**Comment by toiletgranny** [comment]
@dmtrjsg @bedeho Alright, all Gleev logos are ready to be used.

1. All Gleev & Gleev Studio logos, along with my earlier explorations and other variations which might be helpful for future designers in case a further work on the icon is required, can be found [in this Figma file](https://www.figma.com/file/RBKQyC2bs8VWqm0mkY4kJv/Gleev-brand?node-id=85%3A21186),
2. All Gleev & Gleev Studio logos are exported to JPG, PNG, PDF, and SVG files, in different sizes, colors, and layout versions and can be found [in the Design repository in the "logos" folder](https://github.com/Joystream/design/tree/master/logo). I also included a Blender file with a 3D model of the logo, in case that's helpful for future agencies.
3. I took the liberty of exporting the Pioneer logo in the same way, so all Pioneer logos can now be found in [the same "logos" folder inside of the Design repository.](https://github.com/Joystream/design/tree/master/logo)


<img width="1185" alt="CleanShot 2022-10-11 at 15 15 12@2x" src="https://user-images.githubusercontent.com/35307309/195100900-9cb0e2d6-9b86-458b-8bee-0f78f1b2a527.png">
